src/main/java/com/axiope/service/cfg/BaseConfig.java

```diff
@@ -1278,6 +1279,18 @@ public IRepository zenodoRepository() {
     return rc;
   }
 
+  /**
+   * Creates a new {@link IRepository} bean for Digital Commons Data.
+   *
+   * @return the repository adapter
+   */
+  @Bean(name = "digital_commons_dataRepository")
```

Comment:
let's stick to standard camelCase when naming beans, so `DigitalCommonsDataRepository`

Reply:
The name of the bean is automatically created by the configuration property `digital_commons_data.available`
The actual object it is actually called `DigitalCommonsDataRepository` bu the bean name, that is automatically derived from the configuration file, will be `digital_commons_dataRepository`

If I need to change the bean name then I need to change the configuration and the database entry.
Working on it 🧑‍🏭